scripts/development/README.md

```diff
@@ -0,0 +1,29 @@
+
+### Introduction
+
+Hephaestus is a Golang service used to create container images. This document will guide you through the steps required to set up and run Hephaestus locally.
+
+### Prerequisites
+Before starting the setup process, ensure that you have access to the Hephaestus repository, Minikube installed on your system, and have set the environment variables as follows:
+`WEBHOOK_SERVER_CERT_DIR=/<your-path-to>/hephaestus/local-development/webhook-certs`
+
+### Setup Process
+In the Hephaestus repository, run go mod tidy to ensure that all dependencies are installed.
+Set the necessary webhook environment variable by adding the following program arguments: start -c local-development/hephaestus.yaml.
+Run the setup scripts available under the scripts/development directory. The scripts need to be run in the following order:
+1. `00-sanity-check.sh`: This script asserts that required executables are installed on the host.
+2. `01-setup-minikube.sh`: This script creates a Minikube cluster with adequate resources and an extended port range. Note that if you encounter the error "GUEST_DRIVER_MISMATCH," remove the --driver=hyperkit argument on line 16 in the Minikube start command.
```

Comment:
since we made the code change, we probably don't need this?
```
Note that if you encounter the error "GUEST_DRIVER_MISMATCH," remove the --driver=hyperkit argument on line 16 in the Minikube start command.
```